### DISCUSSION TITLE
Integration of React & node

### DISCUSSION THREAD

**Discussion by Ld-star** [community/community#39442] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a frontend made with react.js and a backend made with node.js
What should I do to startup the node server and execute the React(frontend)?

**Comment by shdwkl** [comment, marked as answer]
Pretty solid chance it's `npm start` from the project root.

Properly packaged modules will have some [node scripts](https://docs.npmjs.com/misc/scripts) configured in `package.json`. It's customary to use `start` as the script to run the dev environment, though some might use `build`, `dev`, or other names.

**Reply by Ld-star** [reply]
sure, I know it.
I want to Integration of React & node.
and I want to know to put React build in node.js
Please help me
